docs/content/docs/recipes/collaborative-text-area.md

```diff
@@ -320,28 +322,35 @@ if (sharedString) {
 
 ## Start the Fluid server and run the application
 
-In the Command Prompt, run the following command to start the Fluid service. Note that `tinylicious` is the name of the Fluid service that runs on localhost.
+1. In the Command Prompt, run the following command to start the Fluid service. Note that `tinylicious` is the name of the Fluid service that runs on localhost.
 
-```dotnetcli
-npx tinylicious
-```
+    ```dotnetcli
+    npx tinylicious
+    ```
 
-Open a new Command Prompt and navigate to the root of the project; for example, `C:\My Fluid Projects\collaborative-text-area-tutorial`. Start the application server with the following command. The application opens in your browser. This may take a few minutes.
+    If tinylicious is not installed, you will be prompted to install it. When the Fluid service is running, you will see `info: Listening on port ...` in the Command Prompt.
 
-```dotnetcli
-npm run start
-```
+1. Open a new Command Prompt and navigate to the root of the project; for example, `C:\My Fluid Projects\collaborative-text-area-tutorial`. Start the application server with the following command. The application opens in your browser. This may take a few minutes.
```

Comment:
Nit ```suggestion
1. Open a new Command Prompt and navigate to the root of the project; for example, `C:\My Fluid Projects\collaborative-text-area-tutorial`. Start the application server with the following command. The application will open in your browser. This may take a few minutes.
```

I'm also curious about the "This may take a few minutes" note. Do we expect that to be true? That seems like quite the wait.

Reply:
I don't know the origin of that remark, but it's not needed and probably wrong; so I'll delete it.

Reply:
```suggestion
1. Open a new Command Prompt and navigate to the root of the project; for example, `C:\My Fluid Projects\collaborative-text-area-tutorial`. Start the application server with the following command. The application opens in your browser.
```